### PR TITLE
lesson-3

### PR DESCRIPTION
**Pull request by MaryiaTS**
Made new branch called 'lesson-3'. Added metatags  with content. Added an image. Added a list. Added a table. added a form considering all tasks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,147 @@
...
+    <meta name="description" content="homework" />
+    <meta name="keywords" content="HTML, tags" />
+    <meta name="author" content="Mariya Tsimashkova" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lesson 3</title>
+  </head>
+  <body>
+    <h1>Lesson 3</h1>
+    <hr />
+    <img src="./image/blueflower.jpg" alt="blue flower" />
```
Comment (5tronciy) [inline index.html]: The image is absent.

File: index.html [hunk 2]
```diff
@@ -1,10 +1,147 @@
...
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="homework" />
+    <meta name="keywords" content="HTML, tags" />
+    <meta name="author" content="Mariya Tsimashkova" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (5tronciy) [inline index.html]: Trailing slash on void elements has no effect.

File: index.html [hunk 3]
```diff
@@ -1,10 +1,147 @@
...
+    <title>Lesson 3</title>
+  </head>
+  <body>
+    <h1>Lesson 3</h1>
+    <hr />
```
Comment (5tronciy) [inline index.html]: Trailing slash on void elements has no effect.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

File: index.html (hunk 1 above)
Comment (5tronciy) [reply index.html]: Trailing slash on void elements has no effect.

File: index.html [hunk 4]
```diff
@@ -1,10 +1,147 @@
...
+            name="name"
+            placeholder="Name"
+            required
+          /><br />
+          <label for="email">Email</label><br />
+          <input id="email" type="email" name="email" required /><br />
+          <label for="password">Password</label><br />
+          <input id="password" type="password" name="password" required /><br />
+          <label for="level">Your programming level</label><br />
+          <input id="level" type="range" name="level" min="0" max="10" />
```
Comment (5tronciy) [inline index.html]: Look carefully at the task.

File: index.html [hunk 5]
```diff
@@ -1,10 +1,147 @@
...
+          <label for="http-3">Hyper Text Transfer Protocol</label><br />
+          <label for="comment">Comment</label><br />
+          <textarea id="comment" name="comment" maxlength="500"></textarea><br />
+          <label for="select_html">What does HTML stand for?</label><br />
+          <select name="select_html" id="select_html">
+            <option disabled selected>Your answer</option>
+            <option value="1">Hyperlinks and Text Markup Language</option>
+            <option value="2">Home Tool Markup Language</option>
+            <option value="3">Hyper Text Markup Language</option>
+          </select>
```
Comment (5tronciy) [inline index.html:141]: 'What does http stand for' is missed.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
There are a lot of unnecessary slashes. I haven't attended all of them, but you should check all tags and fix, if necessary.

**Comment by 5tronciy** [comment]
And don't forget to request a review after the task is ready.

**Comment by MaryiaTS** [comment]
> There are a lot of unnecessary slashes. I haven't attended all of them, but you should check all tags and fix, if necessary.

Эти слэши ставятся в непарных тегах при использовании prettier, но в этом проекте я вручную их уберу

**Comment by 5tronciy** [comment]
> > There are a lot of unnecessary slashes. I haven't attended all of them, but you should check all tags and fix, if necessary.
> 
> Эти слэши ставятся в непарных тегах при использовании prettier, но в этом проекте я вручную их уберу

Yep, Prettier do that :D
Anyway, it is not a mistake, and the code will work the same. Just attend you that the trailing slash doesn't influence a document.
[Here](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-in-void-element-start-tags-do-not-mark-the-start-tags-as-self-closing) you may read more about it.

**Review by 5tronciy — APPROVED** [review]
Good work!
Approve.

**Pull request merged**